rend3/src/renderer/mod.rs

```diff
@@ -442,6 +457,46 @@ impl Renderer {
         );
     }
 
+    /// Sets the joint positions for a skeleton. See
+    /// [Renderer::set_skeleton_joint_deltas] to set the vertex transformations
+    /// directly, without having to supply two separate matrix vectors.
+    ///
+    /// ## Inputs
+    /// - `joint_global_positions`: Contains one transform matrix per bone,
+    ///   containing that bone's current clobal transform
+    /// - `inverse_bind_poses`: Contains one inverse bind transform matrix per
+    ///   bone, that is, the inverse of the bone's transformation at its rest
+    ///   position.
+    #[track_caller]
+    pub fn set_skeleton_joint_transforms(
+        &self,
+        handle: &SkeletonHandle,
+        joint_global_positions: &[Mat4],
+        inverse_bind_poses: &[Mat4],
+    ) {
+        let joint_deltas: Vec<Mat4> = joint_global_positions
```

Comment:
It would be nice to factor both this and the other code in some way to get some code re-use. It's not a major deal, but I think would improve things a bit.

Reply:
I agree. I added an associated function to the `Skeleton` type which combines the matrices together and this is now used both in the Skeleton constructor and in the renderer